tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -40,7 +40,6 @@ var _ = Describe("odo devfile create command tests", func() {
 
 	Context("Enabling experimental preference should show a disclaimer", func() {
 		It("checks that the experimental warning appears for create", func() {
-			helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
```

Comment:
It has been added in the ```BeforeEach``` step

Reply:
> It has been added in the BeforeEach step. So i am removing the same occurrence in other places too.

can you please check the same situation in rest of the devfile test scripts too?

Reply:
Ahh...give me a moment

Reply:
Fortunately we have no such occurrence in rest of test files :)

Reply:
/lgtm